test/unit/helpers/people_helper_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class PersonHelperTest < ActionView::TestCase
+  include ApplicationHelper
+
+  test "it disambiguates names with trailing spaces" do
+    person_one = build(:person, forename: "Peter ", surname: "Jones")
```

Comment:
The people have to have the same name to trigger the disambiguation call. I stubbed the response as otherwise I'd have had to create a load of other objects that get involved, `Role`, `RoleAppointment` probably a load of other stuff. The return value of `#name_with_disambiguator` is insignificant. We only care that `disambiguated_people_names` returns the values in its results if the `Person`s have the same name but one has a trailing space. The test is testing that the names are considered the same really. Perhaps I could have named it more clearly.